liquidhaskell-boot/src/Language/Haskell/Liquid/Types/Specs.hs

```diff
@@ -425,6 +425,7 @@ data Spec lname ty = Spec
   , dsize      :: ![([F.Located ty], lname)]                          -- ^ Size measure to enforce fancy termination
   , bounds     :: !(RRBEnvV lname (F.Located ty))
   , axeqs      :: ![F.EquationV lname]                                -- ^ Equalities used for Proof-By-Evaluation
+  , defines    :: ![(F.LocSymbol, ([F.Symbol], F.Expr))]              -- ^ Logic aliases
```

Comment:
This should be `[(F.LocSymbol, ([F.Symbol], F.ExprV lname))]` or names are not going to be resolved in the rhs of defines.